pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -162376,6 +162374,27 @@ self: {
        broken = true;
      }) {};
 
+  "megaparsec_7_0_5" = callPackage
+    ({ mkDerivation, base, bytestring, case-insensitive, containers
+     , criterion, deepseq, mtl, parser-combinators, scientific, text
+     , transformers, weigh
+     }:
+     mkDerivation {
+       pname = "megaparsec";
+       version = "7.0.5";
+       sha256 = "0bqx1icbmk8s7wmbcdzsgnlh607c7kzg8l80cp02dxr5valjxp7j";
+       libraryHaskellDepends = [
+         base bytestring case-insensitive containers deepseq mtl
+         parser-combinators scientific text transformers
+       ];
+       benchmarkHaskellDepends = [
+         base containers criterion deepseq text weigh
+       ];
+       description = "Monadic parser combinators";
+       license = stdenv.lib.licenses.bsd2;
+       hydraPlatforms = stdenv.lib.platforms.none;
+     }) {};
```

Comment:
@peti I know we normally don't commit these changes to `hackage-packages.nix` in PRs, but I was thinking that the daily automatic update should just overwrite this, so there won't be any sort of problems with merge conflicts or anything.

If I was wrong about this (and there does end up being problems), please let me know.